utils/logging.py

```
@@ -0,0 +1,28 @@
+import os
+import sys
+
+from loguru import logger
+
+
+def setup_loguru(log_dir="logs"):
```

Ruff (0.11.9)

Missing return type annotation for public function `setup_loguru`

Add return type annotation: `None`

(ANN201)